src/main/resources/application.yaml

```diff
@@ -32,7 +33,7 @@ security:
     - "/error"
     - "/favicon.ico"
   ### Update list with service id of any service going to interact with casse document api
-  authorisedServices: ${CASE_DOCUMENT_S2S_AUTHORISED_SERVICES:ccd-case-document-am-api,ccd_gw,xui_webapp}
+  authorisedServices: ${CASE_DOCUMENT_S2S_AUTHORISED_SERVICES:ccd_case_document_am_api,ccd_gw,xui_webapp}
```

Comment:
Do we need to have self Id for the authorised list of services?

Reply:
No, We don't need it. I have removed it.